src/core/inline-idl-parser.js

```diff
@@ -169,6 +169,7 @@ function renderAttribute(details) {
   const { parent, identifier, renderParent } = details;
   const { identifier: linkFor } = parent || {};
   const html = hyperHTML`${renderParent ? "." : ""}<a
+      data-link-type="idl"
```

Comment:
Is this documented anywhere?

Reply:
https://tabatkins.github.io/bikeshed/#link-types

Reply:
@sidvishnoi, this could get more specific, per BikeShed/Shepherd. Like IDL-name or whatever.

Reply:
There are lots of them: https://tabatkins.github.io/bikeshed/#dfn-types (Links have the same types as definitions)
I'm not sure if being this explicit will be helpful for? It's good in core/inline-idl-parser though, as we can remove `data-xref-type`

Reply:
Will work on this part after #2854